packages/babel-plugin-transform-classes/src/builtin-classes.js

```diff
@@ -9,7 +9,7 @@ export default new Set([
   "Float32Array",
   "Float64Array",
   "Function",
```

Comment:
I'm pretty sure having at least `HTMLElement` in this list by default would make many people happy

Reply:
I'm okay with it, but I'd like to wait for other opinions (since currently Babel is only about ECMAScript, and not about DOM extensions)

Reply:
I think it's a reasonable exception to the rule, specially considering Babel is used the most for the Web.

Reply:
I would also add `Event` and `CustomEvent`. Considering that these classes aren't wrapped until used, this doesn't seem to add more cost to the transform. I think this transform will be most popular with web components users, so having a useful default seems better to me.